data/mods/innawood/recipes/other/parts.json

```
@@ -262,4 +262,4 @@
     "qualities": [ { "id": "SAW_M", "level": 1 } ],
     "components": [ [ [ "steel_chunk", 1 ], [ "scrap", 3 ], [ "knife_butter", 2 ] ] ]
   }
-]
+]
```

```suggestion
]

```
File ends need a newline the bot's not broke